h2o-py/tests/testdir_algos/infogram/pyunit_PUBDEV_9085_cv_test_weights.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import print_function
+import os
+import sys
+
+sys.path.insert(1, os.path.join("..","..",".."))
+import h2o
+from h2o.estimators.infogram import H2OInfogram
+from tests import pyunit_utils
+    
+def test_cv_warning_messages():
+    """
+    In this test, I will enable cv, add weights to training dataset and the validation
+    dataset.  There should be no warning on weight columns generated
+    """
+    fr = h2o.import_file(path=pyunit_utils.locate("smalldata/admissibleml_test/Bank_Personal_Loan_Modelling.csv"))
+    target = "Personal Loan"
+    fr[target] = fr[target].asfactor()
+    x = ["Experience","Income","Family","CCAvg","Education","Mortgage",
+         "Securities Account","CD Account","Online","CreditCard"]
+    weight = pyunit_utils.random_dataset_real_only(fr.nrow, 1, misFrac=0)
+    weight = weight.abs()
+    weight.set_name(0, "weight_column")
+    fr = fr.cbind(weight)
+
+    splits = fr.split_frame(ratios=[0.80])
+    train = splits[0]
+    test = splits[1]
+    infogram_model_cv_v = H2OInfogram(seed = 12345, protected_columns=["Age","ZIP Code"], nfolds=3) 
```

Comment:
Shouldn't we set `weight_column` parameter when creating `H2OInfogram` in some of the tests?

Reply:
Yes, I have set weight_column in some tests. That is why I have three tests:

1. No weight columns for training or validation frame, cv on.  This should not generate any warning on weigth_column: pyunit_PUBDEV_9085_cv_without_weights.py
2. Weight column for training but no weight column for validation frame, cv on.  This should generate warning on weight column found found in validation frame.  There should be no warning on _internal_cv_weights: pyunit_PUBDEV_9085_cv_test_no_weights.py
3. Weight column for training and validation frame, cv on.  Should not generate any warning messages.  pyunit_PUBDEV_9085_cv_test_weights.py.

Reply:
Absolutely!